network-api/networkapi/wagtailpages/tests/buyersguide/test_products.py

```diff
@@ -239,18 +239,30 @@ def test_product_with_enabled_category_shows_cta(self):
         Testing that a product with any assigned category that has
         show_cta=True displays the CTA as expected.
         """
+        self.bg.call_to_action = buyersguide_factories.BuyersGuideCallToActionFactory()
+        self.bg.save()
+
         product_page = self.product_page
 
         cat1 = BuyersGuideProductCategory.objects.create(name="Cat 1", show_cta=True)
-        category_orderable = ProductPageCategory(
+        cat2 = BuyersGuideProductCategory.objects.create(name="Cat 2", show_cta=False)
+        category_orderable_1 = ProductPageCategory(
                 product=product_page,
                 category=cat1,
             )
-        category_orderable.save()
-        self.product_page.product_categories.add(category_orderable)
+        category_orderable_2 = ProductPageCategory(
+            product=product_page,
+            category=cat2,
+        )
+        category_orderable_1.save()
+        category_orderable_2.save()
+
+        self.product_page.product_categories.add(category_orderable_1)
+        self.product_page.product_categories.add(category_orderable_2)
         self.product_page.save_revision().publish()
 
         response = self.client.get(product_page.url)
+
         self.assertEqual(response.status_code, 200)
         self.assertIsNotNone(response.context['featured_cta'])
```

Comment:
Can you add some context as to why we are testing only the case for two categories on a product, as opposed to having also tests for products with only one category?

Reply:
Updated to add tests for single categories!